ja/api/configuration-generate.md

```diff
@@ -210,7 +229,7 @@ false を設定した場合、ルーティングパスに従う形で HTML フ
 -----| item.html
 ```
 
-_Note: このオプションは、 [Netlify](https://netlify.com) をはじめとする、 HTML によるフォールバックを利用する静的サイトホスティングサイトにおいて利用されます。_
+_Note: このオプションは、[Netlify](https://netlify.com) をはじめとする、 HTML によるフォールバックを利用する静的サイトホスティングサイトにおいて利用されます。_
```

Comment:
```suggestion
_情報: このオプションは、[Netlify](https://netlify.com) をはじめとする、 HTML によるフォールバックを利用する静的サイトホスティングサイトにおいて利用されます。_
```
Could you translate `note` into Japanese, please?

Reply:
Thank you! 😄